lib/widgets/screens/sign_in/sign_in_screen.dart

```diff
@@ -64,161 +65,342 @@ class _SignInScreenState extends State<SignInScreen> {
-                        mainAxisSize: MainAxisSize.max,
-                        crossAxisAlignment: CrossAxisAlignment.center,
-                        children: [
-                          TextFormField(
-                              key: const Key('emailTextField'),
-                              controller: emailController,
-                              decoration: const InputDecoration(
-                                  labelText: "Email Address"),
-                              validator: (value) {
-                                bool validEmail = _validateEmail(value!);
-                                if (validEmail != true) {
-                                  return 'Please enter a valid email';
-                                }
-                                return null;
-                              },
-                              obscureText: false),
-                          SizedBox(
-                            height: mediaQuery.height * 0.01,
-                          ),
-                          TextFormField(
-                              key: const Key('passwordTextField'),
-                              controller: passwordController,
-                              decoration:
-                                  const InputDecoration(labelText: "Passsword"),
-                              validator: (value) {
-                                if (value == null || value.isEmpty) {
-                                  return 'Please enter your password.';
-                                }
-                                return null;
-                              },
-                              obscureText: true),
-                          Padding(
-                            padding: const EdgeInsetsDirectional.fromSTEB(
-                                0, 20, 0, 0),
-                            child: ElevatedButton(
-                              key: const Key('btnSignIn'),
-                              onPressed: () async {
-                                final router = GoRouter.of(context);
-                                final scaffoldManager =
-                                    ScaffoldMessenger.of(context);
+                    Center(
+                      child: SizedBox(
+                        width: Dimensions.loginBoxWidth,
+                        child: Column(
+                          mainAxisSize: MainAxisSize.max,
+                          crossAxisAlignment: CrossAxisAlignment.center,
+                          children: [
+                            SizedBox(
+                              height: mediaQuery.height * 0.03,
+                            ),
+                            TextFormField(
+                                key: const Key('emailTextField'),
+                                controller: emailController,
+                                decoration: const InputDecoration(
+                                  labelText: "Email or phone number",
```

Comment:
Hi Miguel. Great catch. I have mentioned this issue in the PR as well noted in this question: How do you localize text fields within the form, for instance: "Email or phone number" or "password"? The standard l10n.field_name is giving an error, namely "201:52: Error: Not a constant expression." Would love to hear if you have any thoughts. :)

Reply:
Oh I missed that! That happens because you have 'const' right before InputDecoration, and l10n.field_name is not a constant. When you declare an object as constant, you are basically saying that every field under it is also constant. The solution here is to remove 'const' from InputDecoration and apply it only to the subfields that are constant, which won't include labelText. Example:

```
...
decoration: InputDecoration(
        labelText: l10n.field_name,
        border: const OutlineInputBorder(),
),
...
```

Reply:
Ah, that makes sense. Thanks, Miguel. Will make changes accordingly.